README.md

```diff
@@ -64,16 +64,17 @@ The binary has the following options:
 
 ```sh
 Flags:
-  -c, --chart-search-root string      "directory to search recursively within for charts (default ".")"
```

Comment:
Sorry @Grraahaam I am copying directly from the `-h` output here. Have you added the quotes manually?